.github/workflows/ci.yml

```
@@ -13,6 +13,7 @@ on:
       - test/*
       - chore/*
       - build/*
+      - snyk-fix-*
```

```suggestion
      - snyk-*
```

see https://github.com/digita-ai/dgt-shared/pull/183 for example